base/applications/mscutils/eventvwr/evtdetctl.c

```diff
@@ -838,31 +838,30 @@ EventDetailsCtrl(HWND hDlg, UINT uMsg, WPARAM wParam, LPARAM lParam)
                 case IDC_NEXT:
                 {
                     BOOL bPrev = (LOWORD(wParam) == IDC_PREVIOUS);
-                    INT iItem, iSel;
+                    INT iItem, iSel, nItems = ListView_GetItemCount(hwndListView);
+                    WCHAR szText[200];
+
+                    if (nItems <= 0) /* No items? */
+                    {
+                        LoadStringW(hInst, IDS_NOITEMS, szText, _countof(szText));
```

Comment:
I don't think a message box is necessary or wanted: this creates unnecessary action from the user to go and click ok.
Just doing nothing (running the `break;`) would be enough.
In addition, these previous/next buttons should be disabled if there are no items (in the InitDetailsDlgCtrl function)

Reply:
Fixed.